WindowsServerDocs/storage/refs/refs-overview.md

```diff
@@ -99,6 +99,21 @@ Deploying ReFS as a backup target is best suited for applications and hardware t
 > [!NOTE]
 > Backup targets include the above supported configurations. Please contact application and storage array vendors for support details on Fiber Channel and iSCSI SANs. For SANs, if features such as thin provisioning, TRIM/UNMAP, or Offloaded Data Transfer (ODX) are required, NTFS must be used.
 
+### Performance considerations
+
+Pro:
+ReFS is an advanced filesystem over NTFS in many ways. It can greatly improve the performance in read / write operation scenarios. This is especially the case when comparing operations like creating Hyper-V virtual disks, snapshots or merging them. These performance benefits are not limited to Hyper-V though. 
+
+Based on real world applications this also applies for other containered data structures that are similar to the technology of VHDX, such Storage Spaces, especially when mirrored, or S2D which uses mirroring as per concept, aswell as FSLogix containers. This does also apply to Citrix PVS images, to name a 3rd party application scenario.
+
+Con:
+While **supported**, there exist scenarios where ReFS performance might not meet expectations compared to NTFS formatted volumes.
```

Comment:
```suggestion
While **supported**, there exist scenarios where ReFS performance may not meet the expectations, compared to NTFS formatted volumes.
```
Reason: Use "may" instead of "might" when there is a probability for something to happen.